### PR TITLE
Various improvements to seednode service / install scripts

### PR DESCRIPTION
**Pull request by wiz**
- Add setting in seednode configuration to specify btcnode host/port        
- Tweak seednode torrc configuration options to improve P2P connectivity        
- Require bitcoin.service from bisq-seednode.service via systemd binding        
- Make seednode installer run from master and build bisq from release tag
- Seednode must be shutdown using `kill -9` until #3884 is fixed
- Fix seednode uninstall script to use correct service names

### CODE REVIEW

**Comment by wiz** [comment]
Tested OK on Ubuntu 18.04 LTS

File: seednode/torrc [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+
+SafeSocks 0
+HiddenServiceStatistics 0
+AvoidDiskWrites 1
+
+MaxClientCircuitsPending 64
+KeepalivePeriod 2
+#CircuitBuildTimeout 5
+#NewCircuitPeriod 15
+#NumEntryGuards 8
```
Comment (ripcurlx) [inline seednode/torrc]: @wiz  As I saw some messages in the explorer channel regarding tor settings. Is this torrc configuration still final?

Comment (wiz) [reply seednode/torrc]: Okay yeah I just pushed a commit which comments out the advanced settings for now, I will address this in a separate PR

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**